tests/ut/ir/test_compile_pipeline.py

```
@@ -0,0 +1,147 @@
+        name="extra",
+    )
+    with passes.PassContext([outer_instrument], verification_level=passes.VerificationLevel.NONE):
+        result = _run_pass_pipeline(
+            _scalar_program(),
+            operation="lower",
+            extra_instruments=(extra_instrument,),
+        )
+    assert isinstance(result.transformed_program, ir.Program)
+    assert seen
+    assert len(seen) % 2 == 0
+    for outer_event, extra_event in zip(seen[::2], seen[1::2]):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)